project/Common.scala

```diff
@@ -39,6 +41,11 @@ object Common extends AutoPlugin {
     javacOptions in (Compile, compile) ++=
       // From jdk9 onwards this is covered by the '-release' flag above
       onlyOnJdk8("-target", "1.8"),
+
+    // in test code we often use destructing assignment, which now produces an exhaustiveness warning
+    // when the type is asserted
+    scalacOptions in (Test, compile) += "-P:silencer:globalFilters=match may not be exhaustive",
```

Comment:
I added this. Do you of any other way of avoiding these, @raboof?

Reply:
No, this seems reasonable to me!